app/assets/src/components/visualizations/heatmap/Heatmap.js

```diff
@@ -54,6 +54,8 @@ export default class Heatmap {
         maxRowClusterWidth: 100,
         maxColumnClusterHeight: 100,
         spacing: 10,
+        metadataAddLinkHeight: 14,
+        colorHoverLink: "#2b52cd", // used for filter to make icon blue
```

Comment:
This is the value for $primary-medium, not $primary-light. Typo?

Reply:
Also, I don't think it's necessary to make `colorHoverLink` an option. Since the blue color is hard-coded in CSS for the line and the "ADD METADATA" label, there's not currently a situation where you want to allow the user to pass in a different color (like red) here. You can just make a constant at the top of the file.

Reply:
When I input the value for `$primary-light`, the icon actually appeared noticeably lighter than the text and the line. I tried looking into this a little bit but couldn't find a clear answer, so I used `$primary-medium` as a workaround for the time being since I think it matches slightly better. It's likely that I don't fully understand how the filter color matrix works, so I'm still looking into that.

(Pictures below for reference.)

`$primary-light`:
![image](https://user-images.githubusercontent.com/53838890/65181886-e94f4c00-da14-11e9-9c88-a20bc345ac43.png)
`$primary-medium` (current):
![image](https://user-images.githubusercontent.com/53838890/65181889-ebb1a600-da14-11e9-9196-3f396a754687.png)

Reply:
If you decide to move it to a constant, maybe use the ability of exporting values: https://github.com/css-modules/css-modules/blob/master/docs/values-variables.md
This way we can make sure the values are in sync with css.

Reply:
Found the answer to my color struggles in a comment [here](https://stackoverflow.com/questions/21977929/match-colors-in-fecolormatrix-filter) btw 😄I had to add an attribute `color-interpolation-filters="sRGB"` to the filter.

The icon color should match now!

![image](https://user-images.githubusercontent.com/53838890/65186043-9f1e9880-da1d-11e9-8d94-f10a4816877b.png)

Reply:
I think Tiago's link only lets you import css variables into other css files. It can be hard to import css into js unless you do some kind of CSS-in-JS solution (which involves using a completely different framework)

Reply:
Does adding color-interpolation-filters="sRGB" to your <feColorMatrix> help? I think it gets the color a lot closer, but still not perfect.

It's a little scary that feColorMatrix doesn't work as advertised...it seems to produce approximate color values for some reason. I looked into the spec and wasn't able to find a reason.

Reply:
@MarkAZhang No, the `@value` is used to export to js and css.
See, accepted answer here: https://github.com/css-modules/css-modules/issues/106#issuecomment-211607047

Reply:
I was able to export the value with `:export` following the example [here](https://github.com/css-modules/css-modules/issues/86#issuecomment-158273243) (Thanks a ton to Tiago for finding this for me!)

Reply:
I'd never heard of this! Very cool.